extractors/personal_identifiers/location_extraction/code_snippet_common.md

```diff
@@ -0,0 +1,35 @@
+```python
+import spacy
+from typing import List, Tuple
+
+def location_extraction(text: str, label: str) -> List[Tuple[str, int]]:
+    """
+    @param text: the input text
+    @param label: the label that is assigned to extracted words
```

Comment:
what is label used for?

Reply:
This is the `extraction_keyword` that is also used in all the other extraction bricks, I just misnamed it, sorry.